crowbar_framework/app/models/api/upgrade.rb

```diff
@@ -158,6 +161,18 @@ def upgrade_controller_nodes
         # FIXME: save the global status information that this node is being upgraded
         node_api.upgrade
 
+        # Explicitly mark node1 as the cluster founder
+        # this is one time action only (for each cluster)
+        Api::Pacemaker set_node_as_founder drbd_slave
+
+        # Remove "pre-upgrade" attribute from node1
+        # We must do it from the node where pacemaker is running
+        master_node_api = Api::Node.new drbd_master
+        master_node_api.remove_pre_upgrade_role_for drbd_slave
```

Comment:
Maybe it would still be better to execute this action on the node where the role actually is present. But we need running pacemaker for that. So we might do it before we stop pacemaker, before the node is upgraded (in `node_api.upgrade`). However, we'd have to make sure services won't migrate to this node once we remove `pre-upgrade` attribute... so maybe we should put the node in the standby mode before stopping pacemaker and upgrading it?